pkg/operation/botanist/component/logging/kuberbacproxy/kube_rbac_proxy.go

```diff
@@ -50,8 +50,8 @@ type Values struct {
 	Namespace string
 }
 
-// NewKubeRBACProxy creates a new instance of kubeRBACProxy for the kube-rbac-proxy.
-func NewKubeRBACProxy(so *Values) (component.Deployer, error) {
+// New creates a new instance of kubeRBACProxy for the kube-rbac-proxy.
+func New(so *Values) (component.Deployer, error) {
```

Comment:
```suggestion
func New(client client.Client, namespace string) (component.Deployer, error) {
```
After `Values` struct was removed (see previous comment).